src/supermarket/app/controllers/api/v1/cookbook_versions_controller.rb

```diff
@@ -112,4 +117,13 @@ def require_collaborator_params
     params.require(:collaborator_failure)
     params.require(:collaborator_feedback)
   end
+
+  def create_metric(cookbook_version, quality_metric, failure, feedback)
+    MetricResult.create!(
+      cookbook_version: cookbook_version,
+      quality_metric: quality_metric,
+      failure: failure,
+      feedback: feedback
+    )
+  end
```

Comment:
To avoid some duplication within the #foodcritic_evaluation and #collaborators_evaluation methods.  I'm fine either doing this in each controller or doing it within a sub method like this, seemed a little dryer.